Sources/SwiftRepo/Repository/Store/SwiftDataStore.swift

```diff
@@ -88,6 +97,7 @@ public class SwiftDataStore<Model: StoreModel>: Store where Model: PersistentMod
     private func evict(for key: Key) throws {
         let predicate = Value.predicate(key: key)
         try modelContext.delete(model: Value.self, where: predicate)
+        try modelContext.save()
```

Comment:
Creating an issue to work on seeing what is needed to remove these saves without issue.